testdata/src/a/a.go

```diff
@@ -58,4 +59,7 @@ UPDATE * FROM test WHERE test=?`
 	f5 := `
 UPDATE * ` + `FROM test` + ` WHERE test=?`
 	_ = db.QueryRow(f5, s) // want "Use Exec instead of QueryRow to execute `UPDATE` query"
+
+	err := errors.New("oops")
```

Comment:
Do you need this code?

Reply:
Yes, it's the test case that proves that the fix work, it prevents regression.